ADOPTERS.md

```diff
@@ -0,0 +1,12 @@
+ To add your organization to this list, simply add your organization's name,
+ optionally with a link. The list is in alphabetical order.
+
+# Organizations Using Helm
+
+- [Blood Orange](https://bloodorange.io)
+- [Microsoft](https://microsoft.com)
+- [IBM](https://www.ibm.com)
+- [Qovery](https://www.qovery.com/)
+- [Samsung SDS](https://www.samsungsds.com/)
+
```

Comment:
```suggestion
[Ville de Montreal](https://montreal.ca)
```

Reply:
I tried the chrome extension to add my sign-off, but I don't believe it works for suggestions.
Here is the sign off:
Signed-off-by: Marc Khouzam <marc.khouzam@montreal.ca>